MrAaronOlsen/trail-up
# Guest hompage

#### What does  this PR do?

Feature test for user can visit homepage. Additionally adds method to determine which partial to display on navbar.

#### Where should the reviewer start?

New method on application controller, otherwise run rspec.

#### How should this be manually tested?

Ensure when running server that you don't see "Create an Event" until you are logged in

#### Any background context you want to provide?

n/a

#### What are the relevant tickets?

User can visit homepage

#### Screenshots (if appropriate)
#### Questions:
  - Do Migrations Need to be ran?  no
  - Do Environment Variables need to be set? no
  - Any other deploy steps? no
